docs/data/external-data/ga-mtpi/_overview.md

```diff
@@ -0,0 +1,3 @@
+## About
+
+A Multi-scale topographic position image of Australia has been generated by combining a topographic position index and topographic ruggedness. Topographic Position Index (TPI) measures the topographic slope position of landforms. Ruggedness informs on the roughness of the surface and is calculated as the standard deviation of elevations. The derived ternary image captures variations in topographic position across these spatial scales (blue local, green intermediate and red regional) and gives a rich representation of nested landform features that have broad application in understanding geomorphological and hydrological processes and in mapping regolith and soils over the Australian continent.
```

Comment:
```suggestion
A multi-scale topographic position image of Australia has been generated by combining a topographic position index with topographic ruggedness. The Topographic Position Index (TPI) measures the topographic slope position of landforms. Togographic ruggedness informs on the roughness of the surface and is calculated as the standard deviation of elevations. The derived ternary image captures variations in topographic position across these spatial scales (blue local, green intermediate, and red regional) and gives a rich representation of nested landform features that have broad application in understanding geomorphological and hydrological processes and in mapping regolith and soils over the Australian continent.
```

Reply:
I'm fine with this, but I think it comes down to whether we want to adopt the external provider's wording/content exactly as they have written it with no further modifications (e.g. copy-paste content from their portal), or whether we want to customise it on our end.

Reply:
Oh.. I didnt realise it was a copy-paste. In that case, I'll leave it as-is. I think it's ok to: A) substantially rewrite it, B) keep it exactly as-is.